spec/index.bs

```diff
@@ -3726,25 +3726,25 @@ GPURenderPipeline includes GPUPipelineBase;
 
 <script type=idl>
 dictionary GPURenderPipelineDescriptor : GPUPipelineDescriptorBase {
-    GPUVertexStateDescriptor vertexState = {};
-    required GPUProgrammableStageDescriptor vertexStage;
-    GPUPrimitiveAssemblyStateDescriptor primitiveAssemblyState = {};
-    GPURasterizerStateDescriptor rasterizerState = {};
-    GPUProgrammableStageDescriptor fragmentStage;
-    GPUDepthStencilStateDescriptor depthStencilState;
-    required GPUOutputMergerStateDescriptor outputMergerState;
+    GPUVertexInputState vertexInput = {};
+    required GPUProgrammableStage vertexStage;
```

Comment:
The wording above calls everything a "stage" and calls vertex/fragment specifically "shaders". Should this, too? (This could affect compute pipelines too.)

Reply:
it would be unfortunate to say `GPUprogrammableShader`...
Perhaps, what we have here is fine? Non-programmable stages have "state", but programmable stages have shaders.
Would you see `required GPUProgrammableStage vertexShader` as an improvement?

Reply:
I think it would be fine, but I also think "vertexStage" is fine.